src/scripts/convert_lovd2.php

```diff
@@ -0,0 +1,1346 @@
+            ' will be linked.', 'text', 'transcriptid', 10),
+        array('', '', 'note', 'Click the transcript in the table above to copy its ID here.'),
+        'skip',
+        array('Select LOVD2 export file to convert', '', 'file', 'LOVD2_export', 50),
+        array('', 'Current file size limits:<BR>LOVD: ' . ($nMaxSizeLOVD/(1024*1024)) .
+            'M<BR>PHP (upload_max_filesize): ' . ini_get('upload_max_filesize') .
+            '<BR>PHP (post_max_size): ' .
+            ini_get('post_max_size'), 'note', 'The maximum file size accepted is ' .
+            round($nMaxSize/pow(1024, 2), 1) . ' MB' . ($nMaxSize == $nMaxSizeLOVD? '' :
+                ', due to restrictions on this server. If you wish to have it increased, contact' .
+                ' the server\'s system administrator') . '.'),
+        array('Character encoding of imported file', 'If your file contains special characters ' .
+            'like &egrave;, &ouml; or even just fancy quotes like &ldquo; or &rdquo;, LOVD needs ' .
+            'to know the file\'s character encoding to ensure the correct display of the data.',
+            'select', 'charset', 1, $aCharSets, false, false, false),
+        array('', '', 'note', 'Please only change this setting in case you encounter problems ' .
+            'with displaying special characters in imported data. Technical information about ' .
+            'character encoding can be found <A ' .
+            'href="http://en.wikipedia.org/wiki/Character_encoding" target="_blank">on Wikipedia' .
+            '</A>.'),
+        'skip',
+        array('', '', 'print', 'User IDs in the selected LOVD2 export file are usually different' .
+            ' from those in the LOVD3 application. Below one can define a "Fixed user ID" to ' .
+            'set all user IDs in the file to a single value. One can also specify a translation ' .
+            'between LOVD2 and LOVD3 IDs. The translation has precedence over the fixed value. ' .
+            'If both fields are left empty, the user IDs are left untouched.'),
+        array('Fixed submitter ID', 'All user IDs in the imported file will be replaced with ' .
+            'this value. (E.g. all IDs in the edited_by, created_by fields)', 'text',
+            'submitterid_fixed', 10),
+        array('', '', 'note', 'All user IDs in the imported file will be replaced with this ' .
+            'value. (E.g. all IDs in the edited_by, created_by fields)'),
+        array('Submitter ID translation table', '', 'textarea', 'submitterid_translation', 20, 6),
+        array('', '', 'note', 'Translation table for user IDs. On every line an LOVD2 user ID ' .
+            'is expected, followed by an LOVD3 user ID, separated by whitespace. Any submitters ' .
+            'present in the selected file will translated according to this table.'),
+        array('Fixed curator ID', '', 'text', 'curatorid_fixed', 10),
```

Comment:
It would be good to let this field have some explanation too, like the fixed submitter ID.